docs/user_guide/pdks.rst

```diff
@@ -24,7 +24,7 @@ An example of some of the fundamental settings are shown below.
     chip.set('pdk', process, 'lvs', 'runset', <tool>, <stackup>, <runset_type> <file>)
     chip.set('pdk', process, 'devmodel', <tool>, <modeltype>, <stackup>, <file>)
 
-To support standard RTL2GDS flows, the PDK setup will also need to specify pointers to routing technology rules, layout abstractions, layer maps, and routing grids as shown in the below example. For a complete set of available PDK parameters, see the :ref:`Schema<SiliconCompiler Schema>`. ::
+To support standard RTL2GDS flows, the PDK setup will also need to specify pointers to routing technology rules, layout abstractions, layer maps, and routing grids as shown in the below example. For a complete set of available PDK parameters, see the :keypath:`pdk` section of the :class:`.Schema`. ::
```

Comment:
:Class:`schema` does nowhere, I think a :ref: is okay for that one

Reply:
Looks like:

    :ref:`Schema <SiliconCompiler Schema>`